components/Pages/BondingActions/hooks/useTransaction.tsx

```diff
@@ -132,8 +133,10 @@ export const useTransaction = () => {
         )
       } else if (data.bondingAction === ActionType.withdraw) {
         return withdrawTokens(client, senderAddress, data.denom, config)
-      } else {
+      } else if (data.bondingAction === ActionType.claim) {
         return claimRewards(client, senderAddress, config)
+      } else {
+        return createNewEpoch(client, config, senderAddress)
```

Comment:
what about bonding action Actiontype.createEPOCH? 
Looks fine but curious

Reply:
cause execution message is new_epoch, just wanted to be consistent